tests/e2e/commerce/agreement/conftest.py

```diff
@@ -0,0 +1,64 @@
+import datetime as dt
+
+import pytest
+from dateutil.relativedelta import relativedelta
+from freezegun import freeze_time
+
+
+def get_iso_dt_str(dt_obj: dt.datetime) -> str:
+    """Convert datetime to ISO 8601 string with UTC normalization."""
+    dt_obj = dt_obj.astimezone(dt.UTC)
+    return dt_obj.isoformat(timespec="milliseconds").replace("+00:00", "Z")
+
+
+@pytest.fixture
+def invalid_agreement_id():
+    return "AGR-0000-0000"
+
+
+@pytest.fixture
+def agreement_factory(  # noqa: WPS211
+    account_id,
+    seller_id,
+    buyer_id,
+    licensee_id,
+    commerce_product_id,
+    authorization_id,
+):
+    @freeze_time("2025-11-14T09:00:00.000Z")
+    def _agreement_factory(  # noqa: WPS430
+        name: str = "E2E Created Agreement",
+        client_external_id: str = "test-client-external-id",
+        vendor_external_id: str = "test-vendor-external-id",
+    ):
+        """Factory to create agreement data."""
```

Comment:
Docstrings are not needed in test so it can be removed 

<img width="575" height="305" alt="Image" src="https://github.com/user-attachments/assets/eacf42c9-ea11-4443-949b-f680bf92f226" />